webpack.config.js

```diff
@@ -90,29 +91,6 @@ module.exports = {
       ignoreOrder: false, // Enable to remove warnings about conflicting order
       filename: './[name].min.css'
     }),
-    new RemovePlugin({
```

Comment:
There's no point removing these, they don't affect performance as they're not used, and probably could be used to improve our CSS efficiency.

Reply:
I added it back again to reduce the scope of the PR.